src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1,5 +1,15 @@
 Microsoft.CodeAnalysis.Diagnostics.AnalysisContext.RegisterOperationAction(System.Action<Microsoft.CodeAnalysis.Diagnostics.OperationAnalysisContext> action, params Microsoft.CodeAnalysis.OperationKind[] operationKinds) -> void
+Microsoft.CodeAnalysis.Diagnostics.AnalysisResult
```

Comment:
I'm not a fan of exposing this type and having two ways to get diagnostics from CompilationWithAnalyzers - however i don't have any great suggestions either :(